forge/forge/tvm_to_python.py

```diff
@@ -810,6 +810,44 @@ def populate_vstack_args(graph, nid, compiler_cfg):
     args = [("slices", f"{slice_size}")]
     return args
 
+def populate_unsqueeze_args(graph, nid, compiler_cfg):
+    node = graph["nodes"][nid]
+    input_shape = graph["nodes"][node["inputs"][0][0]]["attrs"]["shape"][0][0]
+    output_shape = node["attrs"]["shape"][0][0]
+    
+    assert len(input_shape) + 1 == len(output_shape), "Unsqueeze only supports adding 1 dimension"
```

Comment:
Do we maybe have dim as part of the node attributes? 

If yes, we can simplify this check and below logic for determining which dim is requested. Same applies for squeeze args as well.